tower-batch-control/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "tower-batch-control"
-version = "0.2.41-beta.16"
+version = "1.0.0-beta.17"
```

If we want to bump this to 1.0.0-beta.0, that seems okay too.

```suggestion
version = "0.2.41-beta.17"
```

---

That was a bug. Thanks for noticing it. I'll revert it.

---

I force-pushed a fix.